src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,227 @@
+          Repeat: 1
+          Threads: 1
+          Collection: *collection
+          Operations:
+            - OperationName: drop
+
+  - Name: CreateTimeSeriesCollection
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          Operation:
+            OperationMetricsName: CreateTimeSeriesCollection
+            OperationName: RunCommand
+            OperationCommand:
+              { create: *collection, timeseries: { timeField: "time" } }
+
+  - Name: InsertData
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          CollectionCount: 1
+          Threads: 1
+          DocumentCount: 1000000
+          BatchSize: 1000
+          Document:
+            # Since step = 900 ms and documentCount = 1,000,000, the first and last document
+            # would be 900,000,000 ms apart (roughly 10 days).
+            time: { ^IncDate: { start: "2022-01-01", step: 900 } }
+            temp: { ^RandomDouble: { min: -30, max: 120 } }
+            # This is the value we are going to query against. We will have ~1600 buckets in this
+            # collection and roughly 600 documents per bucket. By making this an increasing value, we can write
+            # predicates to target certain number of buckets.
+            measurement0: { ^Inc: { start: 1 } }
+            # The fields below create larger documents to stress unpacking.
+            measurement1: { ^RandomDouble: { min: -100.0, max: 900.00 } }
+            measurement2: { ^RandomString: { length: 10 } }
+            measurement3:
+              { ^Array: { of: { ^Choose: { from: [1, 2, 3] } }, number: 6 } }
+            measurement4: { ^RandomString: { length: 15 } }
+            measurement5: { ^RandomString: { length: 20 } }
+            measurement6: { ^RandomString: { length: 5 } }
+
+  - Name: Quiesce
+    Type: QuiesceActor
+    Threads: 1
+    Database: *database
+    Phases:
+      OnlyActiveInPhases:
+        Active: [3]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+
+  # Run queries that target a different number of buckets.
+  - Name: SimpleMatchMeasurement10Buckets
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [4]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                # With 600 documents per bucket, 10 buckets will have 'measurement0' <= 6000 (600 * 10).
+                # We'll add 100 just to be safe.
```

Comment:
It's not clear what we are trying to be safe from. If you want to _guarantee_ exactly 10 buckets being selected, let's test it does happen for whatever number is chosen in the filter.